arm64/kallisto/Dockerfile

```diff
@@ -0,0 +1,43 @@
+# (c) Copyright 2022 Lucyfer Contributors
+
+# vim: set fileencoding=utf-8:
+
+
+# TODO:  Implement buildx in a safe manner
+# docker buildx use lucyfer-builder
+FROM            --platform=linux/arm64 ubuntu:jammy
+MAINTAINER      lucyfer AT cime.net
+
+
+USER            root
+ENV             LANG en_US.UTF-8 LANGUAGE en_US:en LC_ALL en_US.UTF-8
+
+RUN             DEBIAN_FRONTEND=noninteractive && \
+                rm /bin/sh && ln -s /bin/bash /bin/sh && \
+                apt-get update && \
+                apt-get -y upgrade && \
+                apt-get -y --no-install-recommends install locales software-properties-common && \
+                locale-gen ${LC_ALL} 
+
+RUN             DEBIAN_FRONTEND=noninteractive && \
+                apt-get -y --no-install-recommends install \
+                    curl \
+                    default-jdk \
```

Comment:
jammy's default jdk is basedon on OpenJDK v11 but I'm not sure anything beyond v8 is safe to use with blazegraph.  Also not even sure ubuntu's OpenJDK is safe to use with blazegraph

Reply:
Latest Blazegraph RC is supposed to work all the way to Java latest from Oracle.